### ISSUE TITLE
[Feature Request] Show Last Modified metadata & Go Up one level

### ISSUE THREAD

**Issue by Sidefix** [halverneus/static-file-server#29]
Hi!

After using this server for more, I've come to the realization of how incredibly useful the Last Modified file/folder metadata information is, and I wish that this could be displayed somehow on the file listing, if possible.

Second major request is if, like on FTP servers for example, an Up One Level line could be added to the top of every page, so that a user could go up one level intuitively, without having to fiddle with the browser Back button.

Thanks in advance for your review!

**Comment by halverneus** [comment]
I don't think that will be an issue. When I get some free time I'll dig in a bit deeper. With this, I'm also considering adding in a JSON response if the client does a request for JSON content.

**Comment by Sidefix** [comment]
After even more usage I think **size** metadata, in addition to last modified date, would also be of great use.

**Comment by aguiraf** [comment]
any update?

**Comment by halverneus** [comment]
I'm afraid not. Between work and family, I haven't had any time to work on new features, sadly. I would like to say I will get to it in the near future, but I don't think that is true, given my elevated role at work. If anyone wants to put in the labor, I'll definitely accept a merge request.

**Comment by colthreepv** [comment]
the responsible for this is the default implementation of `http.serveFile`, [used here](https://github.com/halverneus/static-file-server/blob/8ee2db13a3f62854af894a06b74208266bfbf92b/cli/server/server.go#L38)

I am not a proficient Go coder, but ideally if any library or framework exposes a function compatible with that signature:
```go
func(http.ResponseWriter, *http.Request, string)
```

it would be very easy to swap the default `http.serveFile` out with a config option
